### ISSUE TITLE
TypeError on urldecode()

### ISSUE THREAD

**Issue by cocochepeau** [middlewares/referrer-spam#3]
At line 65 of `ReferrerSpam.php`, `$referrer` can sometimes be null. This causes a TypeError exception:

```
TypeError
urldecode() expects parameter 1 to be string, null given
```

https://github.com/middlewares/referrer-spam/blob/master/src/ReferrerSpam.php#L65

**Comment by oscarotero** [comment]
Hey, thanks!
Fixed and released the patch 2.0.1

**Comment by cocochepeau** [comment]
Perfect! Thanks for the quick fix. :)
